pandablocks/__init__.py

```diff
@@ -1,3 +1,13 @@
 from pandablocks._version_git import __version__
 
-__all__ = ["__version__"]
+from . import asyncio, blocking, commands, connections, hdf, responses
```

Comment:
You can revert this file, as the namespaces of the files below are explicit, non-overlapping, and optional, so it makes more sense to import them directly

Reply:
Resolved in https://github.com/PandABlocks/PandABlocks-client/pull/24/commits/fb7549f5f49429b98f7de220b6b326adea40e442

Reply:
Properly fixed this in https://github.com/PandABlocks/PandABlocks-client/pull/24/commits/58fe8a36d22b161e8d547c2e26925d9492315843